src/sentry/web/client_config.py

```diff
@@ -233,10 +232,7 @@ def get_client_config(request=None):
         (serialized_user,) = user_service.serialize_many(
             filter={"user_ids": [user.id]},
             serializer=UserSerializeType.SELF_DETAILED,
-            auth_context=AuthenticationContext(
-                auth=getattr(request, "auth", None),
-                user=request.user,
-            ),
+            as_user=request.user,
```

Comment:
Would like to verify that there wasn't another intended use of the `AuthenticationContext` downstream. Unless I'm mistaken, we were using it only to unpack the user ([`filter_query.py`, lines 108-109](https://github.com/getsentry/sentry/pull/48758/files#diff-c1ebb751d54bd9f33a541208e128b5dcc80ecc88ff92ba256aa0767f6d2c9880L108-L109)).